### DISCUSSION TITLE
Codespaces pro access for students

### DISCUSSION THREAD

**Discussion by dhanush-2501** [community/community#65096] [Codespaces]
### Select Topic Area

Question

### Body

How to access the codespaces pro as a part of the student developers pack? I couldnt able to get it.
any help is appreciated.

**Comment by arafajs** [comment]
- Once your student status is verified, you will see the "Codespaces Pro" offer under the "Benefits" section.
- Click on the "Claim" button to claim the offer.

**Comment by dhanush-2501** [comment]
@arafajs I am an verified student but still when I click on this link it just asks me to buy the plan
<img width="425" alt="Screenshot 2023-08-27 at 8 01 18 PM" src="https://github.com/community/community/assets/79148926/7bf37e63-223d-4d23-b718-7cf79fa06347">

**Reply by arafajs** [reply]
1. Go to the repository where you want to create the Codespace.
2. Click on the Code button.
3. Select the Codespaces tab.
4. Click on the Create codespace button.
5. Select the Pro plan.
6. Click on the Create button.
